src/main/webapp/about.jsp

```diff
@@ -37,6 +37,9 @@
       style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
 
       <h1>About the CodeU Chat App</h1>
+      <p>
+        This is a Chat App created by Spring 2018 CodeU Team 31! We are a group of college students from around the East Coast: Ellen, Joyce, Pete, and Ransford. Oh, and we have an awesome PA, Will! Stay tuned to see this Chat App get more come to life over the semester!
```

Comment:
Can you wrap this at about 80 characters? Let's keep line length limited to 80char (but not a hard limit)

This will avoid needing to do horizontal scrolling and be able to consistently view across development environments